web/app/views/applications/new.html.erb

```diff
@@ -27,7 +27,11 @@
 	    </div>
 	    <div class="form-group required<%= ' has-danger' if @form.errors.include?(:github_repo_url) %>">
 		<%= f.label :github_repo_url, 'GitHub repository URL', class: "col-form-label" %>
-		<%= f.text_field :github_repo_url, value: @form.github_repo_url, placeholder: "https://github.com/org/repo.git", class: "form-control" %>
+    <% if current_user&.github_oauth_access_token %>
```

Comment:
Fix the indentation of these new lines so they line up with the surrounding context.